lib/git-manager.js

```diff
@@ -342,30 +340,26 @@ function parseRepo(repo) {
  */
 function getIssueEvents(repo, number) {
    debug("Getting events for issue #%s", number);
-   return rateLimit(github.issues.getEvents)(params({ number }, repo))
-                       .then(paginate);
+   return github.paginate(githubRest.issues.listEventsForRepo, params({ issue_number: number }, repo))
 }
 
 function getIssueComments(repo, number) {
    debug("Getting comments for issue #%s", number);
-   return rateLimit(github.issues.getComments)(params({ number }, repo))
-                       .then(paginate);
+   return github.paginate(githubRest.issues.listComments, (params({ issue_number: number }, repo)));
 }
 
 function getPullReviewComments(repo, number) {
    debug("Getting pull review comments for pull #%s", number);
-   return rateLimit(github.pullRequests.getComments)(params({ number }, repo))
-                .then(paginate);
+   return github.paginate(githubRest.pulls.listReviewComments,params({ pull_number: number }, repo))
 }
 
 function getCommit(repo, sha) {
-   return rateLimit(github.repos.getCommit)(params({ sha }, repo))
-                      .then(res => res.data);
+   return githubRest.repos.getCommit(params({ ref: sha }, repo)).then(res => res.data);
```

Comment:
Is the `res.data` still needed?

Reply:
I'm not sure...I'll look into this.

Reply:
I _think_ this is still needed because we need the data property off the response. Was there a particular reason you thought we don't need it?